### PR TITLE
confirmation message for org-ql-view-refresh

### PR DESCRIPTION
**Pull request by xeijin**
In keeping with behaviour of `org-agenda-redo` and associated commands when refreshing an `org-agenda` buffer. If nothing has changed in an `org-ql` buffer it can sometimes be difficult to determine if the refresh took place at all.

### CODE REVIEW

**Comment by alphapapa** [comment]
Hi,

Thanks for your patience.  I'll post a comment in a review.

File: org-ql-view.el [hunk 1]
```diff
@@ -299,7 +299,10 @@ update search arguments."
     (goto-char (point-min))
     (or (when (search-forward current-line nil t)
           (beginning-of-line))
-        (goto-char old-pos))))
+        (goto-char old-pos))
+        (message "org-ql: view refreshed%s"
+                 (when org-ql-view-title
+                   (concat ": " (substring-no-properties org-ql-view-title))))))
```
Comment (alphapapa) [inline org-ql-view.el]: Is it necessary to put the view title in the message?  Won't the user be looking at the view buffer when he refreshes it?

Comment (xeijin) [reply org-ql-view.el]: Fair point, removed.

**Comment by alphapapa** [comment]
Thanks!

**Pull request closed without merging**